src/convertToObject.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 
-/**
+/*
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleUpgrated = sourceString
+    .split(';')
+    .map((el) => el.split(':'))
+    .reduce((prev, oneStyle) => {
+      if (oneStyle[0] && oneStyle[1]) {
+        return { ...prev, [oneStyle[0].trim()]: oneStyle[1].trim() };
+      }
+
+      return prev;
+    }, {});
+
+  return styleUpgrated;
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to improve code readability. Each method should be on a new line, aligned with the others.